test/repositories/identity.test.ts

```diff
@@ -0,0 +1,75 @@
+/**
+ * @jest-environment node
+ */
+
+import { user } from '../factories/user';
+
+import * as userRepository from '@/repositories/user';
+import * as identityRepository from '@/repositories/identity';
+import { AuthProviders } from '@/constants/enum';
+
+describe('identity:', () => {
+  describe('creates identity', () => {
```

Comment:
Ditto as it should be the method name:

```suggestion
  describe('createIdentity', () => {
```